### ISSUE TITLE
Implement remaining GREATEST and LEAST conditional expressions

### ISSUE THREAD

**Issue by ppong** [lumina-tech/gOOQ#11]
https://github.com/lumina-tech/gOOQ/blob/9d1335ab655da9ee921907e2511ae988e048d889/pkg/gooq/function.go#L136

**Comment by jaxonL** [comment]
`SELECT GREATEST(1, 2, 3);` is valid syntax in mysql; should we touch select when implementing these?

**Comment by ppong** [comment]
I think we shouldn't have to touch `select.go` when implementing it. We just need to implement the `Greatest` function.  Following is an example usage: `gooq.Select(gooq.Greatest(1,2,3))`

**Comment by jaxonL** [comment]
I'm assuming that this means that `Select(Greatest(1, 2, 3))` should be valid gooq usage. If that's the case, it's currently not supported with the simplest implementation of greatest/least.

The following is the test case I ran:
```
{
	Constructed:  Select(Greatest(Int64(1), Int64(2), Int64(3))),
	ExpectedStmt: "SELECT GREATEST($1, $2, $3)",
},
```
and the error I got was `panic: runtime error: invalid memory address or nil pointer dereference [recovered]`. I'm assuming it's because `Select`'s implementation expects a `From` clause.
